appveyor.yml

```diff
@@ -11,13 +11,12 @@ environment:
                         # of 32 bit and 64 bit builds are needed, move this
                         # to the matrix section.
       CONDA_VERSION: "4.1.11"
```

Comment:
Unless you have a good reason to pin this in this package, I suggest to remove the version limitation. We already limit it in ``ci-helpers``, currently to 4.3.6 (as that was the latest when last checked).